plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -174,12 +123,55 @@ func Run(s *options.SchedulerServer) error {
 			},
 		},
 	})
-
-	glog.Fatal("this statement is unreachable")
 	panic("unreachable")
 }
 
-func createConfig(s *options.SchedulerServer, configFactory *factory.ConfigFactory) (*scheduler.Config, error) {
+func startHTTP(s *options.SchedulerServer) {
+	mux := http.NewServeMux()
+	healthz.InstallHandler(mux)
+	if s.EnableProfiling {
+		mux.HandleFunc("/debug/pprof/", pprof.Index)
+		mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+		mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+		if s.EnableContentionProfiling {
+			goruntime.SetBlockProfileRate(1)
+		}
+	}
+	if c, err := configz.New("componentconfig"); err == nil {
```

Comment:
It's difficult to tell, but did you ensure this was the 1st place configz was used in the shuffling.  B4 it was the 1st line before refactoring.